lib/main_common.pm

```diff
@@ -1644,7 +1644,7 @@ sub load_filesystem_tests {
         # kernel module required by thin-LVM
         loadtest 'console/snapper_thin_lvm' unless is_jeos;
     }
-    loadtest 'console/snapper_used_space' if is_sle('15-SP1+');
+    loadtest 'console/snapper_used_space' if (is_sle('15-SP1+') || is_leap('15.1+') || is_tumbleweed);
```

Comment:
To follow idea of TW being default (ask @okurz), let's rewrite it as:
`is_sle('15-SP1+') || (is_opensuse && !is_leap('<15.1'))`